py-polars/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "py-polars"
-version = "0.14.29"
+version = "0.14.31"
```

Shouldn't this be 0.14.30?

```suggestion
version = "0.14.30"
```